troubleshooting/app-does-not-connect-to-remote-point.md

```diff
@@ -0,0 +1,47 @@
+---
+title: App Cannot Connect to a Remote Service via HTTPS When Running on Device
+description: App cannot connect to a web service via HTTPS when deployed on a device and produces an unspecified error.
+page_title: App Cannot Connect to Remote Service via HTPS When Running on Device
+slug: app-cannot-connect-to-remote-service
+tags: connect, ios, web-service
+ticketid: 1061272, 1029681, 1011781, 1006104
+---
+
+## Environment 
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Run on Device</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+</table>
+
+## Description
+When running an app on a device, its requests to a remote point via HTTPS do not work, even though the app works in the device simulator. Upon expecting the error arguments of the failed request, you find `statusText: error` and no error message or response text.
```

Comment:
In this case, the status text itself isn't an error. The error message argument of the request is an empty string.